change/@microsoft-fast-foundation-848a0db1-e5de-4362-b813-cdf19212a980.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Are we considering this breaking? Does this need to be "major"?

Reply:
major is none supported type for change files in this repo. It will fail the build gate